File: 2.0/csharp/calls/sendCalls.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;
using CallfireApiClient.Api.Common.Model.Request;
using CallfireApiClient.Api.Campaigns.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var request = new SendCallsRequest()
        {
            DefaultVoice = Voice.MALE1,
            Fields = "items(id,state,toNumber)",
            CampaignId = 4050600003,
            Recipients = new List<CallRecipient>
            {
                new CallRecipient
                {
                    PhoneNumber = "12135551100",
                    LiveMessage = "Hello, Alice, this is message for live answer",
                    MachineMessage = "Hello, Alice, this is message for answering machine"
                },
                new CallRecipient
                {
                    PhoneNumber = "12135551101",
                    LiveMessage = "Hello, Bob, this is message for live answer",
                    MachineMessage = "Hello, Bob, this is message for answering machine"
                }
            }
        };
       
        IList<Call> calls = client.CallsApi.Send(request);
    }
}

File: 2.0/php/calls/sendCalls.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->sendCalls();
        $request->getOperationConfig()->setHeaderParameters(array("Content-Type" => "application/json"));
        $request->getOperationConfig()->setQueryParameters(array("campaignId" => 4050600003,
                                                                 "fields" => "items(id,state,toNumber)",
                                                                 "defaultVoice" => "MALE1"));
        $body = '[
                    {
                        "phoneNumber": "12135551100",
                        "liveMessage": "Hello, Alice, this is message for live answer",
                        "machineMessage": "Hello, Alice, this is message for answering machine"
                    },
                    {
                        "phoneNumber": "12135551101",
                        "liveMessage": "Hello, Bob, this is message for live answer",
                        "machineMessage": "Hello, Bob, this is message for answering machine"
                    }
                 ]';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
